src/plugins/ResizeReorder/ResizerLine.js

```diff
@@ -101,6 +103,9 @@ class ColumnResizerLine extends React.PureComponent {
         cursorDelta: this.props.initialWidth,
       });
     }
+    this.state.top = this.props.instance.getBoundingClientRect().top;
```

Comment:
You shouldn't mutate state. Use `this.setState` instead. You might then want to compare the state change to skip setState so that you don't run into infinite loops.

Reply:
I need a mutable variable, hence I shall migrate `top`, `left` and `knobWidth` to another variable `instanceDetails`.